exotations/task_maps/task_map/init/Point2Plane.in

```diff
@@ -0,0 +1,9 @@
+extend <exotica/TaskMap>
+
+// inherited from TaskMap:
+// string Link           > name of frame in which point is defined
+// VectorXd LinkOffset   > coordinate of point in Link frame
+// string Base           > name of frame in which line is defined
+// VectorXd BaseOffset   > starting point of line in Base frame
+
+Required exotica::Initializer Plane;
```

Comment:
This will limit only defining one plane per task map.
Can you make this work the same way as #318 ?